app/src/main/java/org/fossasia/pslab/adapters/PerformExperimentAdapter.java

```diff
@@ -0,0 +1,45 @@
+package org.fossasia.pslab.adapters;
+
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentManager;
+import android.support.v4.app.FragmentPagerAdapter;
+
+import org.fossasia.pslab.fragment.ExperimentDocFragment;
+import org.fossasia.pslab.fragment.ExperimentSetupFragment;
+
+/**
+ * Created by viveksb007 on 12/7/17.
+ */
+
+public class PerformExperimentAdapter extends FragmentPagerAdapter {
+
+    private final int PAGE_COUNT = 2;
+    private String[] tabTitle = new String[]{"Experiment Doc", "Experiment Setup"};
+
+    public PerformExperimentAdapter(FragmentManager fragmentManager) {
+        super(fragmentManager);
+    }
+
+    @Override
+    public Fragment getItem(int position) {
+        switch (position) {
```

Comment:
This can be put in an `if else` block right? :)

```java
if (position == 1) {
    return ExperimentSetupFragment.newInstance();
} else {
    return ExperimentDocFragment.newInstance();
}
```